packages/kit/src/utils/url.js

```diff
@@ -1,14 +1,19 @@
 import { BROWSER } from 'esm-env';
 
+/** @constant - Matches a URI scheme (https://www.rfc-editor.org/rfc/rfc3986#section-3.1)
+    @type {RegExp}
+    @default
+*/
```

Comment:
updating the formatting here and removing tags that tsc doesn't know about https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html
```suggestion
/**
 * Matches a URI scheme. See https://www.rfc-editor.org/rfc/rfc3986#section-3.1
 * @type {RegExp}
 */
```